pkg/tools/fswatcher/watcher.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package fswatcher provide common tools for FS watching
+package fswatcher
+
+import "github.com/fsnotify/fsnotify"
+
+// WatchOn creates a FS watcher on given ...paths
+func WatchOn(paths ...string) (*fsnotify.Watcher, error) {
```

Comment:
Can we also move [goroutine](https://github.com/networkservicemesh/cmd-forwarder-sriov/blob/5aee0083a1c36299ec55d9d9f887abcbe5b333ae/local/sdk-sriov/pkg/k8s/manager.go#L128) spawn?
```suggestion
func WatchOn(paths ...string) (*fsnotify.Watcher, error) {
```

Reply:
I've moved select logic. Not sure if we want to create goroutine inside of it.